File: Java/Universidad.java
import java.util.ArrayList;
import java.util.List;

public class Universidad {
    private String nombre;
    private List<Carrera> carreras;
    
    public Universidad(String nombre, List<Carrera> carreras) {
        this.nombre = nombre;
        this.carreras = carreras;
    }
    
    public String getNombre() {
        return nombre;
    }
    
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }
    
    public List<Carrera> getCarreras() {
        return carreras;
    }
    
    public void setCarreras(List<Carrera> carreras) {
        this.carreras = carreras;
    }
    
    public void imprimirCarreras() {
        System.out.println("Carreras de la Universidad " + nombre + ":");
        for (Carrera carrera : carreras) {
            System.out.println("- " + carrera.getNombre());
        }
    }
}

File: Phyton/Universidad.py
class Universidad:
    def __init__(self, nombre, carreras):
        self.nombre = nombre
        self.carreras = carreras
    
    def get_nombre(self):
        return self.nombre
    
    def set_nombre(self, nombre):
        self.nombre = nombre
    
    def get_carreras(self):
        return self.carreras
    
    def set_carreras(self, carreras):
        self.carreras = carreras
    
    def imprimir_carreras(self):
        print("Carreras de la Universidad " + self.nombre + ":")
        for carrera in self.carreras:
            print("- " + carrera.get_nombre())